### PR TITLE
Set the default server port to 8088.

### PR DESCRIPTION
**Pull request by hjafarpour**

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -10,5 +10,6 @@ Pushing images is currently handled via `docker push`, and is not part of the bu
 mvn package -DskipTests  # Build local images
 
 # Build images for a private registry; trailing '/' is required:
-# mvn package -DskipTests -Ddocker.registry=docker.example.com:8080/ -Ddocker.tag=$VERSION-$BUILD_NUMBER
+# mvn package -DskipTests -Ddocker.registry=docker.example.com:8088/ -Ddocker
```
Comment (rodesai) [inline docker/README.md]: I think 8080 makes more sense here? I guess it doesn't really matter if its just an example

**Comment by apurvam** [comment]
The system tests set the port to 9093 explicitly, so won't be affected by this patch.

**Comment by apurvam** [comment]
Hojjat, can you confirm the method you used for these changes? just a IDE refactor or a grep and replace?

File: docker/README.md (hunk 1 above)
Comment (hjafarpour) [reply docker/README.md]: Yes, It can be either. I can change it to 8080 :)

**Comment by hjafarpour** [comment]
@apurvam I searched for the 8080 in the IDE and then replaced them manually to make sure it won't replace anything unintentionally.

File: docker/README.md (hunk 1 above)
Comment (hjafarpour) [reply docker/README.md]: @rodesai did you see @apurvam 's comment below about system tests?

**Pull request merged**